lib/config-initializer.js

```diff
@@ -44,6 +44,9 @@ function processAnswers(answers) {
     config.rules.quotes = [2, answers.quotes];
     config.rules["linebreak-style"] = [2, answers.linebreak];
     config.rules.semi = [2, answers.semi ? "always" : "never"];
+    if (answers.extendDefault) {
```

Comment:
The recommended rules will not be style rules, just potential errors (see issue description). Like, `no-unused-vars`, for instance.